code/modules/background/defines.dm

```diff
@@ -45,3 +45,45 @@
 #define RELIGION_HIVEPANTHEON "Hive Pantheon"
 #define RELIGION_PREIMMINENNCE "Preimminennce"
 #define RELIGION_PILOTDREAM "Pilot Dream"
+
+//accent defines
+
+#define ACCENT_CETI "Biesellite"
+#define ACCENT_COC "Frontiersfolk"
+#define ACCENT_ELYRA "Elyran"
+#define ACCENT_ERIDANI "Eridanian"
+#define ACCENT_ERIDANIDREG "Eridanian Dreg"
+#define ACCENT_SOL "Solarian"
+#define ACCENT_SCARAB "Scarab Fleets"
+#define ACCENT_SILVERSUN "Silversun"
+#define ACCENT_PHONG "New Hai Phongese"
+#define ACCENT_MARTIAN "Martian"
+#define ACCENT_KONYAN "Konyanger"
+#define ACCENT_LUNA "Lunan"
+#define ACCENT_GIBSON "New Gibsoner"
+#define ACCENT_HIMEO "Himean"
+#define ACCENT_VYSOKA "Vysokan"
+#define ACCENT_VENUS "Venusian Cytherean"
+#define ACCENT_VENUSJIN "Venusian Jintarian"
+#define ACCENT_DOMINIA "Dominian"
+#define ACCENT_JUPITER "Jovian"
+
+#define ACCENT_REPUBICLANSIIK "Republican Siik'maas"
+#define ACCENT_NAZIRASIIK "Nazira Siik'maas"
+#define ACCENT_CREVAN "Crevan Creole"
+#define ACCENT_DASNRRASIIK "Das'nrra Siik'maas:"
+#define ACCENT_HIGHHARRSIIK "High-Harr'masir Siik'maas"
+#define ACCENT_LOWHARRSIIK "Lower-Harr'masir Siik'maas"
+#define ACCENT_AMOHDASIIK "Amohdan Siik'maas"
+#define ACCENT_RURALDELVAHHI "Rural Delvahhi"
+#define ACCENT_NOMADDELVAHHI "Nomadic Delvahhi"
+#define ACCENT_OLDYASSA "Old Ya'ssa"
+
+#define ACCENT_HEGEMON_NOBLE "Noble Hegemonic"
+#define ACCENT_HEGEMON_PEASANT "Peasant Hegemonic"
+#define ACCENT_TRAD_NOBLE "Noble Traditional"
+#define ACCENT_TRAD_PEASANT "Peasant Traditional"
+
+#define ACCENT_SKRELL "Standard Nral'Malic"
+
+#define ACCENT_DIONA "Diona"
```

Comment:
I wonder. Is it a good idea to use these defines, or should we use type names instead? With the name being fetchable from the accent instance. Type names tend to fuck up less.

Reply:
This is how religion and citizenship are handled right now.